src/hooks/Guilds/ether-swr/guild/useGuildRegistry.ts

```diff
@@ -0,0 +1,19 @@
+import useEtherSWR from '../useEtherSWR';
+import GuildRegistry from 'contracts/GuildsRegistry.json';
+
+export const useGuildRegistry = (contractAddress: string) => {
+  const { data: totalGuilds, error } = useEtherSWR(

```

Comment:
I think we should memoize this

Reply:
Yeah, I thought about it too. But this is what I get when I try to do it: 
![image](https://user-images.githubusercontent.com/59825171/159951080-48287fe4-8423-49ef-b261-1804744846ab.png)

Since the hook needs to be called either way and we're not doing an expensive function call I thought it would be fine to leave it as it is. Or do you have another idea?